source/ui/data/VerticalDelegate.js

```diff
@@ -191,7 +207,8 @@
 			metrics.height = this.pageHeight(list, page);
 			metrics.width  = this.pageWidth(list, page);
 			// update the childSize value now that we have measurements
-			this.childSize(list);
+			var oldChildSize = list.childSize;
+			list.isChildSizeUpdated = (oldChildSize == this.childSize(list)) ? false : true;
```

Comment:
Ah ha, that makes sense - I think this will have to remain calculated here, I might have misread the diff. :)